.gitignore

```diff
@@ -36,3 +36,7 @@ env-config.sh
 /calypso-strings.php
 cover.html
 .test.log
+
+.caches/*
+!.caches/.gitkeep
```

Comment:
> (I hope you've seen Jurassic Park)

Of course! I was sad that there was no velociraptor emoji.

> Should it just be .cache? .cache/babel/* for the files maybe?

git doesn't really have support for keeping empty directories. So the above is fine, or we can add a second .gitignore file in the .caches folder that ignores everything but the .gitignore
```
!.gitignore
```